modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -133,6 +133,16 @@
 		"newReference": "HtmlParser _htmlParser",
 		"to": "_htmlParser.extractText"
 	},
+	{
+		"classNames": [
+			"CommerceAddressLocalService",
+			"CommerceAddressLocalServiceUtil"
+		],
+		"from": "updateCommerceAddress(CommerceAddress)",
+		"issueKey": "LPS-186925",
+		"newImports": "com.liferay.portal.kernel.service.ServiceContextThreadLocal",
```

Comment:
@MicaelleSilvaa, isn't mandatory we use the `[]` in the attribute `newImports` like in other checks?

Reply:
Resolved